modules/internal/Atomics.chpl

```diff
@@ -226,6 +226,7 @@ module Atomics {
     proc init_helper(value:bool) {
       pragma "fn synchronization free"
       pragma "local fn" pragma "fast-on safe extern function"
+      pragma "no gpu codegen"
```

Comment:
FWIW I think this is the opposite of what the default should be, in terms of the pragma:
 * I think we should default to having extern procs disable GPU code generation
 * But, there will be certain extern procs where it is OK, and I think we should mark those with a pragma